zerver/lib/actions.py

```diff
@@ -2567,6 +2568,151 @@ def check_schedule_message(
     return do_schedule_messages([send_request])[0]
 
 
+def check_update_message(
```

Comment:
I think we would like to have some tests for this function, as it does a lot of valuable checks which we might want to be there for all forms of message edits.

Reply:
The `check_update_message` has been tested thoroughly in `test_message_edit`. There is no need to create new tests as the PR only extracts it without making significant changes in features.

Reply:
Agreed.